src/features/codeLensProvider.ts

```diff
@@ -28,12 +28,21 @@ class OmniSharpCodeLens extends vscode.CodeLens {
 export default class OmniSharpCodeLensProvider extends AbstractProvider implements vscode.CodeLensProvider {
 
     private _testManager: TestManager;
+    private _options: Options;
 
     constructor(server: OmniSharpServer, reporter: TelemetryReporter, testManager: TestManager)
     {
         super(server, reporter);
 
         this._testManager = testManager;
+        this._checkOptions();
+
+        let configChangedDisposable = vscode.workspace.onDidChangeConfiguration(this._checkOptions, this);
+        this.addDisposables(configChangedDisposable);
+    }
+
+    private _checkOptions(): void {
```

Comment:
Nit--this method doesn't actually check anything. Maybe call it "_updatedCachedOptions" or something?

Reply:
Let's go back and fix that up with `codeActionProvider` at the same time, so that we're consistent. I'll send a follow up PR with that change.